app/models/ui/Sequence.scala

```diff
@@ -27,6 +27,7 @@ case class ExclusiveSequence(text: Text,
                              hint: Option[Text],
                              options: Seq[Text],
                              exclusiveOption: Text,
+                             exclusiveOptionHint: Option[Text],
```

Comment:
Does accessibility allow this to be optional??

Reply:
The optionality arises from the use of regex matching. In practice, owing to the mechanism used to identify exclusive sequences the hint will always be defined. I have kept the optionality as it provides a useful way of distinguishing between exclusive and non-exclusive sequences when rendering the sequence checkboxes.